src/commands/CoreCommands.js

```diff
@@ -108,34 +109,26 @@ class CoreCommands {
          */
 		try {
 			// Get the response from server
-			let response = JSON.parse(await rp(options))
+			const axiosResponse = await axios(options);
 
 			// If there's no change to be displayed, end
-			if (!response.change || Object.keys(response.change).length === 0) { return }
+			if (!axiosResponse.data.change || Object.keys(axiosResponse.data.change).length === 0) { return; }
 
 			// Else refresh the tree, because there's a new change to be displayed
 			this.appsProvider.refresh()
 		}
 		catch (err) {
-			// Parse the error and display it
-			let e = JSON.parse(err.error)
-
-			// If name is undefined and environment version is 2, then try to use APIError message format instead of syntax error
-			if (e.name === undefined && this._environment.version === 2) {
-				vscode.window.showErrorMessage(`${e.message}: ${e.detail ? e.detail[0] : 'No further information available.'}`);
-			} else {
-				vscode.window.showErrorMessage(`${e.name}: ${e.message}`)
-			}
+			showError(err, 'File saving/uploading failed');
+
+			// Mark the document as dirty (Adds a space and remove it)
```

Comment:
@dominikkadera Muzes prosím zkouknout tohle, jestli bez toho přidání bílého znaku '\u200B' to bude fungovat? Myšleno, jestli ten bílý znak němel ještě jiný speciální význam, než označit soubor jako neuložený.

Reply:
Ahoj,
ten bílý znak byl skutečně jen kvůli tomu ukazování souboru jako dirty, a naopak ještě dělal bordel. Pokud to už jde řešit jinak, tak paráda, a nevím o tom, že by se měl používat ještě k něčemu jinému 😁